h2o-automl/src/test/java/hex/ModelMojoWriterTest.java

```diff
@@ -0,0 +1,67 @@
+package hex;
+
+import ai.h2o.automl.targetencoding.BlendingParams;
+import ai.h2o.automl.targetencoding.TargetEncoder;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.Key;
+import water.TestUtil;
+import water.fvec.Frame;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Map;
+
+/**
+ * This test should be moved to h2o-core module once we move all TargetEncoding related classes there as well. 
+ */
+public class ModelMojoWriterTest extends TestUtil implements ModelTestCommons {
+
```

Comment:
should this test also stall for cloud formation?

Reply:
@honzasterba good point! Thanks, I think maybe it should as I'm doing some map reduce transformations. Do we have any conventions on how big cloud should be for testing? Was wondering if there is some good reasoning behind choosing the number of nodes cc @michalkurka